frontend/packages/ceph-storage-plugin/locales/en/ceph-storage-plugin.json

```diff
@@ -186,7 +186,7 @@
   "Node Table": "Node Table",
   "Create Storage Cluster": "Create Storage Cluster",
   "OCS runs as a cloud-native service for optimal integration with applications in need of storage and handles the scenes such as provisioning and management.": "OCS runs as a cloud-native service for optimal integration with applications in need of storage and handles the scenes such as provisioning and management.",
-  "nodesText": "The selected nodes will be labeled with <1>cluster.ocs.openshift.io/openshift-storage=\"\"</1> (unless they are already labeled). {{replica}} of the selected nodes will be used for initial deployment. The remaining nodes will be used by OpenShift as scheduling targets for OCS scaling.",
+  "nodesText": "The selected nodes will be labeled with&nbsp;<1>cluster.ocs.openshift.io/openshift-storage=&quot;&quot;</1>&nbsp;(unless they are already labeled). {{replica}} of the selected nodes will be used for initial deployment. The remaining nodes will be used by OpenShift as scheduling targets for OCS scaling.",
```

Comment:
&nbsp and &quot are still in string when language is English.
![Screenshot from 2021-02-03 15-02-19](https://user-images.githubusercontent.com/7313189/106711052-e1a43d80-6631-11eb-9b65-340781a64e73.png)

Reply:
Thank you @yanpzhan!